<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_mainTitle</class>
 <widget class="QWidget" name="widget_mainTitle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_bgiLogo" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>0</y>
     <width>332</width>
     <height>103</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_promptInfo" native="true">
   <property name="geometry">
    <rect>
     <x>347</x>
     <y>0</y>
     <width>1101</width>
     <height>103</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_date" native="true">
   <property name="geometry">
    <rect>
     <x>1460</x>
     <y>0</y>
     <width>351</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	color:rgb(135, 206, 235);
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label_Version">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>版本:1.0.0.1 Git:aveihfef</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_OpenSoftware">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>开机:2022/10/13 13:15</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_CurrentTime">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>当前:2022/10/13 13:15</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_about">
   <property name="geometry">
    <rect>
     <x>1820</x>
     <y>70</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: rgb(178,224,230);
	border-radius:9px;
}

QPushButton:hover
{
	background-color: rgb(135,206,235);
}

QPushButton:pressed
{
	background-color: rgb(30,144,255);
	color: white;
}

QPushButton:disabled
{
	background-color: rgb(192,192,192);
}</string>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
